exporter/signalfxexporter/go.mod

```diff
@@ -15,6 +15,7 @@ require (
 	go.uber.org/multierr v1.5.0
 	go.uber.org/zap v1.16.0
 	google.golang.org/protobuf v1.25.0
+	gotest.tools v2.2.0+incompatible
```

Comment:
Use testify assert instead of adding new dep.

Reply:
oh nice catch! I got it automatically imported

Reply:
Yeah it's always importing the wrong one for me haha. Major pain.